src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -306,6 +307,10 @@ export const updateValidationKeyAndSave = (
   payload,
   analyticsSectionName,
 ) => async dispatch => {
+  dispatch({
```

Comment:
can we just do this instead rather than making a new action type?
```
dispatch({
    type: ADDRESS_VALIDATION_INITIALIZE,
    fieldName,
  });
```

Reply:
@erikphansen I'd prefer not to as it clears out address validation state - if there is an error, the modal would be empty.

Reply:
nope!

Reply:
tbh I'm not sure why switching to `ADDRESS_VALIDATION_INITIALIZE` just immediately closes the modal but it does :-/